tests/unit/configFile.spec.js

```diff
@@ -6,6 +8,7 @@ const ConfigFile = require("../../src/js/shared/config_file");
 const defaultSettings = require("../resources/defaultSettings.json");
 const encryptedSettings = require("../resources/encryptedSettings.json");
 
+const generateJSONFile = jest.spyOn(ConfigFile.default, "generateJSONFile");
```

Comment:
No need to add an spy to `generateJSONFile` if the spy-related methods will never be called